src/backend/utils/adt/agtype.c

```diff
@@ -4783,7 +4793,7 @@ Datum age_startnode(PG_FUNCTION_ARGS)
     agtype_value *agtv_value = NULL;
     char *graph_name = NULL;
     char *label_name = NULL;
-    graphid graph_id;
+    graphid graph_oid;
```

Comment:
Shouldn't it be 'graph_id'? Since the type `graphid` is not actually a postgres oid, rather age vertex\edge ID?

Reply:
This was a change from 11 to 12. I believe that it was done due to the graph oid being used in a graphid. I will look into these.